src/psyclone/transformations.py

```diff
@@ -1469,11 +1469,15 @@ class ACCParallelTrans(ParallelRegionTrans):
                            ACCDataDirective, ACCEnterDataDirective,
```

Comment:
Now that we let `default_present` be false, I don't think the region must be within a dynamic or static data region (as the compiler could generate the necessary data-movement instructions). Therefore, please could you update the class docstring. We will probably have to update the checks in the code-generation backend too to allow for this case (i.e. if default_present is false then no EnterData or Data region is required).

Reply:
I have updated the example on the docstring, and I have made it API-agnostic and doctest-valid.

The default present is not being validated by the backend/validate_global_constraints, even when set to True (see doctest example). And I don't think this is a change from this PR. Or am I misunderstanding the issues?

Reply:
Great, thank you. I'm surprised we're not validating that. However, it's not the fault of this PR.